examples/medplum-provider/src/components/tasks/TaskBoard.tsx

```diff
@@ -140,14 +139,12 @@ export function TaskBoard(props: TaskBoardProps): JSX.Element {
     handleTaskChange(task).catch(showErrorNotification);
```

Comment:
It looks like this is duplicative now and will kick off two `fetchTasks` calls. It seems like that function is opting out of the client cache, so I expect this to double up the network call. We should probably remove one of these!